tasks/virt-customize/manifests/virt-customize.yaml

```diff
@@ -0,0 +1,54 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: ClusterTask
+metadata:
+  annotations:
+    pvc.params.task.kubevirt.io/kind: PersistentVolumeClaim
+    pvc.params.task.kubevirt.io/apiVersion: v1
+    script.params.task.kubevirt.io/type: script
+    sshAuthorizedKeysSecret.params.task.kubevirt.io/kind: Secret
+    sshAuthorizedKeysSecret.params.task.kubevirt.io/apiVersion: v1
+  labels:
+    task.kubevirt.io/type: virt-customize
+    task.kubevirt.io/category: virt-customize
+  name: virt-customize
```

Comment:
the disadvantage of this task is that it cannot be run across different namespaces. So it can change only disks in the same namespace as the pipeline is running.

the advantage is that no serviceAccount is needed. 

This could be fixed by implementing a CRD and a controller, although that would be a much larger endeavor than implementing a single task. It might be probably better to include this in a CDI which already does similar things.

Reply:
I dont really see a reason to run the task in a different namespace than the pvc it modifies...